content/manager/update-deployment.md

```diff
@@ -267,30 +266,227 @@ node_templates:
+                operation2:
+                    implementation:
+                        plugin2.path.to.module.taskA
+        relationships:
+          - [...]
+            source_interfaces:
+                interface1:
+                    operation1:
+                        implementation:
+                            plugin1.path.to.module.taskB
+plugins:
+    plugin1:
+        [...]
+    plugin2:
+        [...]
+```
+```yaml
+# deployment update blueprint
+node_templates:
+    node1:
+        interfaces:
+            interface1:
+                operation1:
+                    implementation:  # modified operation1 (changed implementation)
+                        plugin1.path.to.module.taskB
+                # removed operation2
+                operation3:  # added operation 3
+                    implementation:
+                        plugin2.path.to.module.taskB
+        relationships:
+          - [...]
+            source_interfaces:
+                interface1:
+                    operation1:
+                        implementation:  # modified operation1 (changed implementation to a different plugin)
+                            plugin2.path.to.module.taskC
+plugins:
+    plugin1:
+        [...]
+    plugin2:
         [...]
 ```
+
+#### Properties
+Properties can be added, removed or modified. Note that overriding a default property value counts as a property modification.
+```yaml
+# original deployment blueprint
+node_templates:
+    node1:
+        type: Cloudify.nodes.Compute
+    node2:
+        type: my_custom_node_type
+        properties:
+            prop1: value1
+```
 ```yaml
 # deployment update blueprint
 node_templates:
-    node2:  # node1 will be uninstalled. node2 will be installed
+    node1:
+        type: Cloudify.nodes.Compute
+        properties:
+            ip: 192.0.2.1  # modified the property by overriding it default
+    node2:
+        type: my_custom_node_type
+        properties:
+            # removed property prop1
+            prop2: value2  # added property prop2
+```
+#### Outputs
+Outputs can be added, removed or modified.
+```yaml
+# original deployment blueprint
+outputs:
+    output1:
+        value: {get_input: inputA}
+    output2:
+        [...]
+```
+```yaml
+# deployment update blueprint
+outputs:
+    output1:
+        value: {get_input: inputB}  # modified the value of output1
+    # removed output2
+    output3:  # added output3
         [...]
 ```
+#### Workflows
+Workflows can be added, removed or modified.
+```yaml
+# original deployment blueprint
+workflows:
+    workflow1: plugin_name.module_name.task1
+    workflow2:
+        [...]
+```
+```yaml
+# deployment update blueprint
+outputs:
+    workflow1:
+        value: plugin_name.module_name.task2  # modified the value of workflow1
+    # removed workflow2
+    workflow3:  # added workflow3
+        [...]
+```
+
+#### Description:
+The description can be added, removed or modified
```

Comment:
seperate the 3 different examples. Maybe add mini-headlines - remove, add, modify